src/types/events/eventT.ts

```diff
@@ -0,0 +1,20 @@
+import * as t from 'io-ts'
+import {eventTypeT} from "./eventTypeT";
+
+const eventRequired = t.type({
+    source_id: t.string,
+    source_type: t.keyof({
+        'data source': null,
```

Comment:
let's use underscores instead of spaces for the source type, stays consistent with how we've named other properties